code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -169,3 +176,7 @@
 
 /mob/living/carbon/human/mob_negates_gravity()
 	return (shoes && shoes.negates_gravity())
+
+/mob/living/carbon/human/proc/ClothesSlowdown()
+	for(var/obj/item/I in list(wear_suit, w_uniform, back, gloves, head))
```

Comment:
Why only these items? Why not loop over all clothing?

Reply:
That was just what's in the original loop, honestly. I didn't think too hard on it when I was writing it back into a proc.